src/lang-promql/complete/promql.terms.ts

```diff
@@ -467,4 +467,25 @@ export const snippets: readonly Completion[] = [
     info: 'Set or replace a label value in an input vector',
     apply: snippet('label_replace(${__input_vector__}, "${__dst__}", "${__replacement__}", "${__src__}", "${__regex__}")'),
   },
+  {
+    label: 'topk(__top_rank_number__, __metric__)',
```

Comment:
Maybe just `__rank__` or `__rank_number__`? Is rank the right term for something like topk/bottomk? But at least the `top`/`bottom` seems a bit redundant.

`__metric__` means either a single time series or a whole group of series with the same metric name, neither of which is precisely the input requirement here. I've used `__input_vector__` for some of the other snippets, maybe keep that in the new snippets too?

Reply:
sounds good to me. I wasn't really confortable with what I put here anyway :). Thanks for your help !